client/src/components/auth/AuthActions.js

```diff
@@ -0,0 +1,37 @@
+import setAuthToken from '../../utils/setAuthToken';
+import jwtDecode from 'jwt-decode';
+import { browserHistory } from 'react-router';
+import axios from '../../utils/api';
+
+export const CHECKIN_ERROR = 'CHECKIN_ERROR';
+export const SET_CURRENT_USER = 'SET_CURRENT_USER';
+
+
+export function setCurrentUser(user) {
+  return {
+    type: SET_CURRENT_USER,
+    user
+  };
+}
+
+export const checkinUserAction = ({ email, password }) =>
+dispatch => axios.post('/users/login', { email, password })
+  .then((response) => {
+    const token = response.data.token;
+    localStorage.setItem('token', token);
```

Comment:
It's not very good practice to do localStorage stuff in this functions. It's better done in your component